exe/kubernetes-deploy

```diff
@@ -12,7 +12,8 @@ bindings = {}
 verbose_log_prefix = false
 
 ARGV.options do |opts|
-  opts.on("--bindings=BINDINGS", "Expose additional variables to ERB templates (format: k1=v1,k2=v2 or JSON)") do |binds|
+  opts.on("--bindings=BINDINGS", "Expose additional variables to ERB templates " \
+    "(format: k1=v1,k2=v2 or JSON)") do |binds|
```

Comment:
I don't mind having this whitespace fix in, but you'll need to 🎩  the executable just in case if we keep this change. We have no test coverage for the executables files.

Reply:
Manually tested kubernetes-deploy executable with option --bindings=BINDINGS and applied two variables in ERA template with success.